test/Conversion/triton_to_tritongpu.mlir

```diff
@@ -1,7 +1,7 @@
 // RUN: triton-opt %s -split-input-file -convert-triton-to-tritongpu=num-warps=2 | FileCheck %s
 
 tt.func @ops() {
-  // CHECK: module attributes {"triton_gpu.num-warps" = 2 : i32} {{.*}}
+  // CHECK: module attributes {"triton_gpu.num-warps" = 2 : i32, "triton_gpu.threads-per-warp" = 32 : i32} {{.*}}
```

Comment:
I ran this test locally, to make this test pass, we need "triton_gpu.threads-per-warp" = 64

Reply:
They should match the architecture's warp size. However I did not run tests related to this file apparently on MI200 so I didn't touch it.

Reply:
I cannot find it either, so I just ran `root@ubb4-rack-07:/workspace/projects/triton/python# ./build/cmake.linux-x86_64-cpython-3.8/bin/triton-opt  ../test/Conversion/triton_to_tritongpu.mlir -split-input-file -convert-triton-to-tritongpu=num-warps=2`, the did a manual check